cpp/daal/include/algorithms/covariance/covariance_distributed.h

```diff
@@ -99,11 +99,156 @@ class DistributedContainer
 
 /**
  * <a name="DAAL-CLASS-ALGORITHMS__COVARIANCE__DISTRIBUTEDCONTAINER"></a>
- * \brief Provides methods to run implementations of the correlation or variance-covariance matrix  algorithm in the distributed processing mode
-          on master node.
+ * \brief Provides defaultDense method to run implementations of the correlation or variance-covariance matrix  algorithm using single-pass computation method that works
```

Comment:
Should be default instead of single-pass.
You can look on batch/online doxygen comments.